tests/test_core.py

```diff
@@ -4301,7 +4278,7 @@ def test_dylink_raii_exceptions(self):
     ''', expected=['special 2.182810 3.141590 42\ndestroy\nfrom side: 1337.\n'])
 
   @needs_dlfcn
-  @no_wasm_backend('wasm backend resolves symbols greedily on startup')
+  @disabled('wasm backend resolves symbols greedily on startup')
```

Comment:
what does "disabled" mean for a test? temporary, or permanent? triaged for importance, or not?

Reply:
From what I can tell this test expects to be able to load the the main module and the side module which is in the `preloadedWasm` list, but with a missing symbol which is then not provided until runtime.

Its kind of like RTLD_LAZY, but for the load time modules.   I don't think we even support RTLD_LAZY for runtime loaded libraries at this point.   

So I guess the question is: do want want to support this at all?  I think probably not, given that nobody is asking for it right now.

If you agree I'll just remove this test.

Reply:
Actually on native platforms, at least on linux, this seems to be the default:

```
         lazy
               When generating an executable or shared library, mark it to tell the dynamic linker to defer
               function call resolution to the point when the function is called (lazy binding), rather than
               at load time.  Lazy binding is the default.
```

So if we want to be compatible I guess we should try to make this work?  

Should I open a bug and link it here?

Reply:
Oh really, I didn't know linux does that... Interesting.

sgtm to file an issue and link here before landing.